src/doc/rustc/src/targets/known-issues.md

```diff
@@ -0,0 +1,7 @@
+# Known Issues
+
+Cross compiling your code to different architectures via the target-feature is generally considered as unsafe. Below a list with known "gotchas".
```

Comment:
There's two point I think should be stressed here to prevent misunderstandings:

- The table of examples is (and always will be) an incomplete list. So, just because the scenario (target-feature, target) the reader is working with doesn't occur on it doesn't mean everything will be fine.
- These issues can only occur when mixing code that has the target-feature enabled with code that doesn't have it enabled. If someone builds *all* code (including the entire standard library) with a set of target-features, there's no danger.